src/iter.rs

```diff
@@ -10,46 +11,50 @@ use std::io;
 #[cfg(all(feature = "core2", not(feature = "std")))]
 use core2::io;
 
-use crate::error::{InvalidCharError, OddLengthStringError};
+use crate::error::InvalidCharError;
 
 #[rustfmt::skip]                // Keep public re-exports separate.
-pub use crate::error::HexToBytesError;
+pub use crate::error::{HexToBytesError, OddLengthStringError};
```

Comment:
clippy warns about `HexToBytesError` and I think we should re-export `InvalidCharError` because it is public in `HexToBytesIter` (returned by `next`).